### ISSUE TITLE
wb_set_col_widths() throws error if base font is set to "Roboto" beforehand

### ISSUE THREAD

**Issue by jploetner** [JanMarvin/openxlsx2#1079]
Hello, I think I encountered a weird bug. If I set the base font to "Roboto" and then try to change the column widths with `wb_set_col_widths()` it throws an error:

``` r
library(openxlsx2)

## Create a new workbook
wb <- wb_workbook()

## Add a worksheet
wb$add_worksheet("Sheet 1")

## set font
wb$set_base_font(font_name = "Roboto")

## set col widths
wb$set_col_widths(cols = c(1, 4, 6, 7, 9), widths = c(16, 15, 12, 18, 33))
#> Error in col_df$width[select] <- as_xml_attr(widths): replacement has length zero
```

<sup>Created on 2024-07-15 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

I have tried a few other fonts, but only "Roboto" led to that error. If I first set the column widths and then set the base font, it works fine.

**Comment by JanMarvin** [comment]
Hi @jploetner , thanks for the report. I'll look into it. I assume this is because we ship no fontwidth for `roboto`. We only have these for the default Windows fonts, precisely from my Windows 10 installation plus the `Aptos` font. If my impression is correct, we'd have to supply a fallback for non predefined fonts.

**Comment by JanMarvin** [comment]
I've pushed a fix to #1080

**Comment by JanMarvin** [comment]
We use the font widths to calculate the correct width per font. Since we do not ship a tool to calculate this on the fly and instead use a list, filled by a C# program, we have to fall back to a default font for other fonts.
